inventory-management/src/main/java/com/smartnerd/model/User.java

```diff
@@ -0,0 +1,79 @@
+package com.smartnerd.model;
+
+import java.util.Date;
+
+public class User {
```

Comment:
follow the java standard rules. this is normal pojo class you should not give database columns.give normal names
like userId, userName,password,sysCreatedDate,.......etc. update class and commit in next commit,